plugin/skel/listener.go

```diff
@@ -61,11 +59,10 @@ func Listen(socket net.Listener, driver Driver, ipamDriver ipamapi.Ipam) error {
 	}
 
 	if ipamDriver != nil {
-		handleMethod(ipamReceiver, "GetDefaultAddressSpaces", listener.getDefaultAddressSpaces)
-		handleMethod(ipamReceiver, "RequestPool", listener.requestPool)
-		handleMethod(ipamReceiver, "ReleasePool", listener.releasePool)
-		handleMethod(ipamReceiver, "RequestAddress", listener.requestAddress)
-		handleMethod(ipamReceiver, "ReleaseAddress", listener.releaseAddress)
+		// FIXME looks like plugins-helpers does not allow a single
+		// mux to serve multiple APIs. This is problematic, since ATM
+		// we only have one socket.
+		ipamapi.NewHandler(ipamDriver).Serve(socket)
```

Comment:
@bboreham this is a showstopper. I can think of only two ways of addressing it: a) listen on separate sockets for net and ipam, or b) get the plugins-helpers API changed.